android/src/main/cpp/NativeProxy.cpp

```diff
@@ -480,6 +480,35 @@ PlatformDepMethodsHolder NativeProxy::getPlatformDependentMethods() {
   auto maybeFlushUiUpdatesQueueFunction =
       bindThis(&NativeProxy::maybeFlushUIUpdatesQueue);
 
+  enum ScreenTransitionCommand {
+    Start = 1,
+    Update = 2,
+    Finish = 3,
+  };
+
+  auto manageScreenTransitionFunction =
+      [=](jsi::Runtime &rt,
+          int command,
+          int stackTag,
+          const jsi::Value &param) -> jsi::Value {
+    if (command == ScreenTransitionCommand::Start) {
+      std::array<int, 2> screenTags = startScreenTransition(stackTag);
+      if (screenTags[0] > 0) {
```

Comment:
Should it be `> 0` or `-1`? I saw that you set them to `-1` on the screens side, or am I messing the variables?